src/main.ts

```diff
@@ -22,49 +24,95 @@ interface ListFunctionsOptions {
   basePath?: string
   config?: Config
   featureFlags?: FeatureFlags
+  parseISC?: boolean
+}
+
+interface AugmentedFunctionSource extends FunctionSource {
```

Comment:
[dust] Currently, this type only differs from `FunctionSource` in that it also includes ISC. Are you thinking it may be used in the future for other things? If we introduce new options to `listFunctions`, would that not be under a different conditional check than `parseISC`? And if so, do you think this should be called something like `FunctionSourceWithISC`?

Reply:
the `AugmentedFunctionSource` type is really the input type to `getListedFunction`. it's `FunctionSource` + whatever may need to be added to assemble the return value of `listFunctions` and `listFunction`. If there was a new flag, let's say `enablePartyMode`, then we can extend `AugmentedFunctionSource` to have an `partyModeResult?: Parteyyy` property, and `getListedFunction` could use that to assemble the return value.